packages/react/types/tests.tsx

```diff
@@ -159,14 +160,14 @@ const anim1 = keyframes`
 
   // TS@next reports an error on a different line, so this has to be in a single line so `test:typescript` can validate this on all TS versions correctly
   // $ExpectError
-  ;<CompWithoutClassNameSupport prop1="test" css={{ backgroundColor: 'hotpink' }} />
+  ;<CompWithoutClassNameSupport prop1="test" css={{ color: 'hotpink' }} />
 
   const MemoedCompWithoutClassNameSupport = React.memo(
     CompWithoutClassNameSupport
   )
   // TS@next reports an error on a different line, so this has to be in a single line so `test:typescript` can validate this on all TS versions correctly
   // $ExpectError
-  ;<MemoedCompWithoutClassNameSupport prop1="test" css={{ backgroundColor: 'hotpink' }} />
+  ;<MemoedCompWithoutClassNameSupport prop1="test" css={{ color: 'hotpink' }} />
```

Comment:
Why this has to be changed? 🤔

Reply:
This line with `backgroundColor` is too long for Prettier. Even if you don't format it on auto-save in your IDE, you need to remember to do `git commit --no-verify`, because there's a `prettier --write` running on pre-commit.

```json
  "lint-staged": {
    "*.{js,ts,tsx,md}": [
      "prettier --write",
      "git add"
    ]
  }
```


Obviously, I kept forgetting about it and failing tests, because the error occurs in different places (on `css` or over entire element) by different TypeScript version.

I can change it to `backgroundColor` if you want, but that property is irrelevant from the perspective of the test, because the css prop is missing, and with `backgroundColor` it was just more unpleasant to contribute to this file.

Reply:
Right - I don't mind the change, was just curious why it was needed. Your explanation makes sense 👍